src/provider/zeebe/properties/PriorityDefinitionProps.js

```diff
@@ -0,0 +1,162 @@
+  const {
+    element
+  } = props;
+
+  if (!is(element, 'bpmn:UserTask')) {
+    return [];
+  }
+
+  return [
+    {
+      id: 'priorityDefinitionPriority',
+      component: Priority,
+      isEdited: isFeelEntryEdited
+    }
+  ];
+}
+
+function Priority(props) {
+  const {
+    element
+  } = props;
+
+  const commandStack = useService('commandStack');
+  const bpmnFactory = useService('bpmnFactory');
+  const translate = useService('translate');
+  const debounce = useService('debounceInput');
+
+  const getValue = () => {
+    return (getPriorityDefinition(element) || {}).priority;
+  };
+
+  const setValue = (value) => {
+    const commands = [];
+
+    const businessObject = getBusinessObject(element);
+
+    let extensionElements = businessObject.get('extensionElements');
+
+    // (1) ensure extension elements
+    if (!extensionElements) {
+      extensionElements = createElement(
+        'bpmn:ExtensionElements',
+        { values: [] },
+        businessObject,
+        bpmnFactory
+      );
+
+      commands.push({
+        cmd: 'element.updateModdleProperties',
+        context: {
+          element,
+          moddleElement: businessObject,
+          properties: { extensionElements }
+        }
+      });
+    }
+
+    // (2) ensure PriorityDefinition
+    let priorityDefinition = getPriorityDefinition(element);
+    const isNullValue = value === null || value === '' || value === undefined;
+
+    if (priorityDefinition && isNullValue) {
+
+      // (3a) remove priority definition if it exists and priority is set to null
+      commands.push({
+        cmd: 'element.updateModdleProperties',
+        context: {
+          element,
+          moddleElement: extensionElements,
+          properties: {
+            values: extensionElements.get('values').filter(v => v !== priorityDefinition)
+          }
+        }
+      });
+
+    } else if (priorityDefinition && !isNullValue) {
+
+      // (3b) update priority definition if it already exists
+      commands.push({
+        cmd: 'element.updateModdleProperties',
+        context: {
+          element,
+          moddleElement: priorityDefinition,
+          properties: { priority: value }
+        }
+      });
+
+    } else if (!priorityDefinition && !isNullValue) {
+
+      // (3c) create priority definition if it does not exist
+      priorityDefinition = createElement(
+        'zeebe:PriorityDefinition',
+        { priority: value },
+        extensionElements,
+        bpmnFactory
+      );
+
+      commands.push({
+        cmd: 'element.updateModdleProperties',
+        context: {
+          element,
+          moddleElement: extensionElements,
+          properties: {
+            values: [ ...extensionElements.get('values'), priorityDefinition ]
+          }
+        }
+      });
+    }
+
+    // (4) commit all updates
+    commandStack.execute('properties-panel.multi-command-executor', commands);
+  };
+
+  const tooltip = <div>
```

Comment:
We keep the tooltips in the respective [provider](https://github.com/bpmn-io/bpmn-js-properties-panel/blob/main/src/contextProvider/zeebe/TooltipProvider.js). Also, the link doesn't point anywhere at the moment. I guess that's because the documentation doesn't exist yet?

Reply:
Hey there, yeah it's because it doesn't exist yet. Raised a follow-up here: https://github.com/bpmn-io/bpmn-js-properties-panel/issues/1073.

Reply:
On a related topic, there will come a time where this method of translating won't be enough. Not being able to move divs around relative to each-other prevents things like links or formatting inside of sentences.

I think something similar to the [Trans component](https://react.i18next.com/latest/trans-component) of react-i18next might make sense. Just floating the idea around.

Reply:
> I think something similar to the [Trans component](https://react.i18next.com/latest/trans-component) of react-i18next might make sense. Just floating the idea around.

Let's keep things simple and not over-engineer. Quoting from the site you mentioned:

![image](https://github.com/user-attachments/assets/9377b050-3a2c-41cf-8ac1-1fb17a938959)